source/macros.tex

```diff
@@ -172,11 +172,11 @@
 
 %%--------------------------------------------------
 %% Macros for funky text
-\newcommand{\Cpp}{\texorpdfstring{C\kern-0.05em\protect\raisebox{.35ex}{\textsmaller[2]{+\kern-0.05em+}}}{C++}\xspace}
-\newcommand{\CppIII}{\Cpp 2003\xspace}
-\newcommand{\CppXI}{\Cpp 2011\xspace}
-\newcommand{\CppXIV}{\Cpp 2014\xspace}
-\newcommand{\opt}{{\ensuremath{_\mathit{opt}}}\xspace}
+\newcommand{\Cpp}{\texorpdfstring{C\kern-0.05em\protect\raisebox{.35ex}{\textsmaller[2]{+\kern-0.05em+}}}{C++}}
+\newcommand{\CppIII}{\Cpp{} 2003}
```

Comment:
You might also consider `\Cpp~2003` if you want to keep 'C++ 2013' from splitting across two lines.